code/src/pages/dashboard/dashboard-products/DashboardProductsPage.tsx

```diff
@@ -3,15 +3,25 @@ import AddIcon from "@mui/icons-material/Add";
 import DashboardTabContainer from "@/layouts/dashboard-layout/components/dashboard-tab-container/DashboardTabContainer";
 
 import ProductsTable from "@/containers/tables/products-table/ProductsTable";
-import { mockProducts } from "@/containers/tables/products-table/ProductsTable.constants";
 
 import AppBox from "@/components/app-box/AppBox";
 import AppButton from "@/components/app-button/AppButton";
 import AppTypography from "@/components/app-typography/AppTypography";
 
 import routePaths from "@/constants/routes";
+import { useLocaleContext } from "@/context/i18n/I18nProvider";
+import { useGetManagerProductsQuery } from "@/store/api/productsApi";
 
 const DashboardProductsPage = () => {
+  const { locale } = useLocaleContext();
+  const { data, isLoading } = useGetManagerProductsQuery({ lang: locale });
```

Comment:
also handle `error` from the hook. It should to be also

Reply:
Added div with error message if we have an error and also a todo to replace it later

Reply:
> Added div with error message if we have an error and also a todo to replace it later

Maybe handle error with `div` will be enough for now